lesson02/assignment/css/style.scss

```diff
@@ -4,3 +4,97 @@
 @import 'settings.responsive';
 @import 'components.content';
 
+.main{
+  background-color: $gray;
+  width: 951px;
+  height: 474px;
```

Comment:
One of the main pillars of responsive design is flexible sizing units. Almost all sizing should use rem or %.

Reply:
Great reminder. Thank you!